scripts/projects-test/utils.ts

```diff
@@ -72,3 +73,5 @@ export async function shEcho(cmd: string, cwd?: string) {
 export function generateFiles(srcFolder: string, target: string) {
   fsExtra.copySync(srcFolder, target, { recursive: true });
 }
+
+export { workspaceRoot };
```

Comment:
re-exporting for convenience, apps using this domain dont have to import anything else